frontend/packages/console-shared/src/components/popper/Popper.tsx

```diff
@@ -25,10 +25,10 @@ class VirtualReference implements PopperJSReference {
       right: x + width,
       top: y,
       width,
-    };
+    } as DOMRect;
```

Comment:
Why is `as` needed here?

Reply:
resolves
`Type '{ bottom: number; height: number; left: number; right: number; top: number; width: number; }' is missing the following properties from type 'DOMRect': x, y, toJSONts(2739)`

Reply:
That seems like a legitimate type error we should address... Either `DOMRect` is the wrong type or we're not specifying all the properties it should have.

Reply:
I'm going to back this change out in the new PR and see what's needed to resolve the type error.